### ISSUE TITLE
Failing to request data on /void-wiki and /warframesConclave-wiki

### ISSUE THREAD

**Issue by OchseGit** [Snekw/Wf#6]
Hi Snekw!
Can you look at these errors again?
Thank you for your support!

**void-wiki**
```
{
   "error":{
      "message":"Failed at requesting data.",
      "data":{
         
      },
      "inner":{
         "message":"Failed to get lua data.",
         "data":{
            
         },
         "inner":{
            
         },
         "stack":"Error: Failed to get lua data.\n    at getLuaObject.then.catch.err (/var/www/Wf/cache.js:110:31)\n    at <anonymous>\n    at process._tickCallback (internal/process/next_tick.js:182:7)",
         "full":{
            "error":"lua5.3: [string \"...\"]:13: module 'Module:Table' not found:\n\tno field package.preload['Module:Table']\n\tno file '/usr/local/share/lua/5.3/Module:Table.lua'\n\tno file '/usr/local/share/lua/5.3/Module:Table/init.lua'\n\tno file '/usr/local/lib/lua/5.3/Module:Table.lua'\n\tno file '/usr/local/lib/lua/5.3/Module:Table/init.lua'\n\tno file '/usr/share/lua/5.3/Module:Table.lua'\n\tno file '/usr/share/lua/5.3/Module:Table/init.lua'\n\tno file './Module:Table.lua'\n\tno file './Module:Table/init.lua'\n\tno file '/usr/local/lib/lua/5.3/Module:Table.so'\n\tno file '/usr/lib/x86_64-linux-gnu/lua/5.3/Module:Table.so'\n\tno file '/usr/lib/lua/5.3/Module:Table.so'\n\tno file '/usr/local/lib/lua/5.3/loadall.so'\n\tno file './Module:Table.so'\nstack traceback:\n\t[C]: in function 'require'\n\t[string \"...\"]:13: in local 'f'\n\t./luaObjectToJson.lua:25: in main chunk\n\t[C]: in ?\n"
         }
      },
      "stack":"Error: Failed at requesting data.\n    at Cache._getRequest.then.catch.err (/var/www/Wf/cache.js:164:29)\n    at <anonymous>\n    at process._tickCallback (internal/process/next_tick.js:182:7)",
      "full":{
         "error":{
            "error":"lua5.3: [string \"...\"]:13: module 'Module:Table' not found:\n\tno field package.preload['Module:Table']\n\tno file '/usr/local/share/lua/5.3/Module:Table.lua'\n\tno file '/usr/local/share/lua/5.3/Module:Table/init.lua'\n\tno file '/usr/local/lib/lua/5.3/Module:Table.lua'\n\tno file '/usr/local/lib/lua/5.3/Module:Table/init.lua'\n\tno file '/usr/share/lua/5.3/Module:Table.lua'\n\tno file '/usr/share/lua/5.3/Module:Table/init.lua'\n\tno file './Module:Table.lua'\n\tno file './Module:Table/init.lua'\n\tno file '/usr/local/lib/lua/5.3/Module:Table.so'\n\tno file '/usr/lib/x86_64-linux-gnu/lua/5.3/Module:Table.so'\n\tno file '/usr/lib/lua/5.3/Module:Table.so'\n\tno file '/usr/local/lib/lua/5.3/loadall.so'\n\tno file './Module:Table.so'\nstack traceback:\n\t[C]: in function 'require'\n\t[string \"...\"]:13: in local 'f'\n\t./luaObjectToJson.lua:25: in main chunk\n\t[C]: in ?\n"
         }
      }
   }
}
```

**warframesConclave-wiki**
```
{
   "error":{
      "message":"Failed at requesting data.",
      "data":{
         
      },
      "inner":{
         "message":"Failed to get lua data.",
         "data":{
            
         },
         "inner":{
            
         },
         "stack":"Error: Failed to get lua data.\n    at getLuaObject.then.catch.err (/var/www/Wf/cache.js:110:31)\n    at <anonymous>\n    at process._tickCallback (internal/process/next_tick.js:182:7)",
         "full":{
            "error":"lua5.3: [string \"local WarframeData = mw.loadData('Module:Warf...\"]:1: attempt to index a nil value (global 'mw')\nstack traceback:\n\t[string \"local WarframeData = mw.loadData('Module:Warf...\"]:1: in local 'f'\n\t./luaObjectToJson.lua:25: in main chunk\n\t[C]: in ?\n"
         }
      },
      "stack":"Error: Failed at requesting data.\n    at Cache._getRequest.then.catch.err (/var/www/Wf/cache.js:164:29)\n    at <anonymous>\n    at process._tickCallback (internal/process/next_tick.js:182:7)",
      "full":{
         "error":{
            "error":"lua5.3: [string \"local WarframeData = mw.loadData('Module:Warf...\"]:1: attempt to index a nil value (global 'mw')\nstack traceback:\n\t[string \"local WarframeData = mw.loadData('Module:Warf...\"]:1: in local 'f'\n\t./luaObjectToJson.lua:25: in main chunk\n\t[C]: in ?\n"
         }
      }
   }
}
```

**Comment by Snekw** [comment]
I'll take a look at why those are broken again during the weekend.

**Comment by Snekw** [comment]
Fixed in a7a2681c075412e07f3f381b91d50b38bc5b8545 and 2b49363c0cdee116d16650c2e46f8ad328bbeb1f. This has been deployed to https://wf.snekw.com.
